src/hidimstat/statistical_tools/p_values.py

```diff
@@ -420,73 +420,3 @@ def two_sided_pval_from_pval(pval, one_minus_pval=None, distribution="norm"):
     )
 
     return two_sided_pval, two_sided_pval_corr
-
-
-def step_down_max_t(stat, permutation_stats):
```

Comment:
Step-down procedures are useful to enhance power when fwer controled is sought. I think it is OK to remove it, but we may want to put it back one day or the other

Reply:
To have a trace of it, I open an issue.